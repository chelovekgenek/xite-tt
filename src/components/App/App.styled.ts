import styled, { createGlobalStyle } from "styled-components"
import { Grid } from "components/commons"

export const GlobalStyles = createGlobalStyle`
  body {
    margin: 0;
    font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
      'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
      sans-serif;
    -webkit-font-smoothing: antialiased;
    -moz-osx-font-smoothing: grayscale;
  }
  #root {
    height: inherit;
  }
`

export const MainLayout = styled.div`
  padding: 1rem;
  height: 100%;
`
export const Col = styled(Grid.Col)`
  height: 100%;
`
