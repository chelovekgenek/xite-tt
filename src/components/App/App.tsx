import React from "react"
import { Provider } from "react-redux"

import { store } from "store"
import { Playlist } from "components/Playlist"
import { Player } from "components/Player"

import { GlobalStyles, MainLayout, Col } from "./App.styled"

export const App: React.FC = () => (
  <Provider store={store}>
    <MainLayout>
      <GlobalStyles />
      <Col span={4}>
        <Playlist />
      </Col>
      <Col span={20}>
        <Player />
      </Col>
    </MainLayout>
  </Provider>
)
