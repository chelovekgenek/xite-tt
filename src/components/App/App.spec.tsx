import React from "react"
import { shallow } from "enzyme"

import { App } from "./App"

describe("App", () => {
  it("renders correctly", () => {
    const comp = shallow(<App />)
    expect(comp).toMatchSnapshot()
  })
})
