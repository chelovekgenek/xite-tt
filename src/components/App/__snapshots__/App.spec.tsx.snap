// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders correctly 1`] = `
<Provider
  store={
    Object {
      "dispatch": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
      Symbol(observable): [Function],
    }
  }
>
  <styled.div>
    <GlobalStyleComponent />
    <Styled(Col)
      span={4}
    >
      <Playlist />
    </Styled(Col)>
    <Styled(Col)
      span={20}
    >
      <Connect(Player) />
    </Styled(Col)>
  </styled.div>
</Provider>
`;
